System/RTETools.cpp

```diff
@@ -190,4 +190,18 @@ namespace RTE {
 		file = 0;
 		return false;
 	}
-}
\ No newline at end of file
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	std::string RoundFloatToPrecision(float input, int precision) {
+		std::stringstream floatStream;
+		floatStream << std::fixed << std::setprecision(precision) << input;
+		std::string outputString = floatStream.str();
+		// Remove trailing zeroes and decimal point.
```

Comment:
I don't think we should remove trailing zeroes here. Just by the technical (i.e. scientific) definition of precision, if I say I want `1.5` with 3 precision, I should get `1.500`. Similarly, 2 precision would be `1.50`, 1 precision would be `1.5`, 0 precision would be `1`.
From a user point of view, this ensures you get exactly what you ask for out of it, instead of sometimes not doing what the method name suggests.

Also, I feel like this might be useful for lua users, wanna expose it in luaman?